pom.xml

```diff
@@ -65,7 +65,7 @@
         <saxon.version>9.9.1-8</saxon.version>
         <log4j.version>2.19.0</log4j.version>
         <junit.version>5.9.2</junit.version>
-        <elasticsearch.version>7.17.24</elasticsearch.version>
+        <opensearch.version>[2.15.0,)</opensearch.version>
```

Comment:
This kind of notation `[xxx,)` result in a non-reproducable builds as every build may or may not use the same version of this library/tool/...  It made it more complex to verify new functionality, reproduce bugs and may introduce changes which are hard to determinate. If an automatic way is needed to update the version tools like dependabot should be used.

Reply:
I support @henning-gerhardt's remark. 
@stweil please use one specific version number instead!

Reply:
I also rebased the PR to git master.